infra/runtests.py

```diff
@@ -49,6 +49,15 @@ def reload_module(mod):
             delattr(mod, attr)
     importlib.reload(mod)
 
+def import_as(file_path, module_name):
+    # Based in example in importlib documentation
+    import importlib.util
+
+    spec = importlib.util.spec_from_file_location(module_name, file_path)
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[module_name] = module
+    spec.loader.exec_module(module)
```

Comment:
This ugly hack pre-loads `labXXX.full.py` and makes it available as `labXXX`, so that when the tests run and actually import `labXXX` it gets the inlined version.